app/templates/base.html

```diff
@@ -66,42 +66,75 @@
     </div>
   </div>
 
+
+
+
+
+
+
   {% block body %}
   {% endblock %}
 
-  <footer>
-    <!-- <h6 class="uppercase white">STRATUS</h6> -->
-    <div class="footer-title-par">
-      <span class="bold-footer uppercase white" style="">STRATUS</span>
-      <span class="thin-footer white"> - KuriusHacks2020</span>
-    </div>
-    <div class="content">
-      <div class="divider"></div>
-      <div class="row u-center">
-        <div class="col">
-          <a href="https://youtu.be/G5uDEDriXFc">
-            <span class="icon">
-              <i class="zmdi zmdi-youtube-play zmdi-hc-2x" aria-hidden="true"></i>
-            </span>
-          </a>
+  <div class="modal modal-large modal-animated--zoom-in" id="bugForm">
+    <a href="#target" class="modal-overlay close-btn" aria-label="Close"></a>
+    <div class="modal-content" role="document" style="width: 60rem; height: 60vh; border-radius: 15px;">
+        <div class="modal-header">
+            <!-- <a href="#components" class="pull-right" aria-label="Close"><span class="icon">X</span></a> -->
         </div>
-        <div class="col">
-          <a href="https://devpost.com/software/tbd-7yxgu1">
-            <span class="icon">
-              <i class="zmdi zmdi-album zmdi-hc-2x" aria-hidden="true"></i>
-            </span>
-          </a>
+        <div class="modal-body">
+          <iframe class="airtable-embed" src="https://airtable.com/embed/shro1W9DcSXja7K5r?backgroundColor=blue" frameborder="0" onmousewheel="" width="100%" height="750" style="background: transparent; border: 1px solid #ccc;"></iframe>
         </div>
-        <div class="col">
-          <a href="https://github.com/TheAvidDev/stratus">
+    </div>
+</div>
+
+
+<footer>
+  <div class="content">
+      <a href="{{ url('index') }}">
+        <img src="{{ static('img/logo.png') }}" class="logo" alt="HackItForward">
+      </a>
+      <div class="row">
+          <div class="col-3 offset-3">
+              <ul class="no-bullets">
+                  <p class="footer-list-title">About</p>
+                  <a href="#">
```

Comment:
```suggestion
                  <a href="/faq">
```
This and the ones below with hard coded urls are going to be flatpages.